docs/competition/team-support/battery-shuffler.md

```diff
@@ -0,0 +1,24 @@
+---
+original:
+  authors: Richard Barlow
+  url: https://bitbucket.org/richardbarlow/sr-comp-team-support-coord/wiki/Battery_Shuffler
+---
+# Battery Shuffler
+
+A battery shuffler is a volunteer that moves batteries, battery chargers, battery charger PSUs and battery charging bags from the arrival desk to the battery charging station on Saturday morning and between the battery charging station and the kit return desk on Sunday afternoon.
+
+## Saturday Morning Shuffling
+Teams arrive on Saturday morning between 9:00 and 9:30. When they arrive we take the following items off of them at the arrival desk:
+
+ * 2x batteries
+ * 1x battery charger
+ * 1x battery charger power supply
+ * 1x battery charging bag
+
+All of these parts need to be carried from the arrival desk (on floor 0) to the battery charging station (on floor 2) where they will be used by us to manage the charging of batteries.
```

Comment:
Locations here are only relevant to newbury

Reply:
fafe178